etc/schema.json

```diff
@@ -6542,52 +6547,68 @@
                                     "additionalProperties": false,
                                     "properties": {
                                         "frag_ignored": {
-                                            "type": "integer"
+                                            "type": "integer",
+                                            "description": "Number of IPv4 fragments ignored due to internal error"
```

Comment:
I see this is only set when `PoolGet` returns `NULL`, so I think this is not an internal error but more like a out of resources error. Needs a closer look.